IntelBluetoothFirmware/IntelBluetoothFirmware.hpp

```diff
@@ -109,6 +111,10 @@ class IntelBluetoothFirmware : public IOService
     };
     BTType currentType;
     uint32_t boot_param;
+    
+    IOLock *mLock;
+    unsigned long mDevicePowerState;
+    SInt32 mOutstandIO;
```

Comment:
unused variable?

Reply:
They are using in "setPowerState" and "powerChangeDone".